Changes

```diff
@@ -1,4 +1,17 @@
-Release history for Perl module Zonemaster::CLI
+Release history for Zonemaster Component Zonemaster-CLI 
+(Perl module Zonemaster::CLI).
+
+v1.1.1 2017-11-02 Public release
+- Changed dependency from Zonemaster to Zonemaster::Engine due
+  to name space change (#43)
+- Fix Commonmark rendering on Github. Replace NBSP with SPACE. (#49)
+- Various updates to packaging (#52) 
+- Changed dependency from Net::LDNS to Zonemaster::LDNS due to
+  name space change (#51) 
+- Updated installation instruction (#53) 
+- Specify smallest version of Locale::TextDomain i Makefile.PL. (#57)
+- Update licensing (#58) 
+
```

Comment:
Maybe add a heading indicating that all these things have been "Fixed" to associate them with the patch level?

I was thinking about suggesting headings for Changed, Removed (major), Added (minor), Fixed, Deprecated (patch), but I never got to it until now.